mnist/README.md

```diff
@@ -23,6 +23,7 @@ Follow the version of the guide that is specific to how you have deployed Kubefl
 
 1. [MNIST on Kubeflow on GCP](#gcp)
 1. [MNIST on Kubeflow on AWS](#aws)
+1. [MNIST on Kubeflow on AZURE](#azure)
```

Comment:
Could you please delete the TOC in the previous section? It duplicates what we have here. I also don't want  extra sub levels under vanilla k8s.

Could you also delete the last item "MNIST on other platforms" since it no longer exists.

Reply:
Done! Removed the auto generated ToC and manually removed that orphaned entry.